show/bgp_quagga_v4.py

```diff
@@ -1,6 +1,71 @@
 import click
+import json
+import re
+from natsort import natsorted
+from tabulate import tabulate
+from natsort import natsorted
+from tabulate import tabulate
 from show.main import *
```

Comment:
>* [](start = 22, length = 1)

Don't import wildcard in prod code.

Reply:
did not changed wildcard import on this PR. Will sync with Joe on this